src/accelerator.rs

```diff
@@ -0,0 +1,87 @@
+use drivers::Driver;
+use motions::Motion;
+
+fn clamp(n: isize, min: isize, max: isize) -> isize {
+  if n < min {
+    return min;
+  }
+  if n > max {
+    return max;
+  }
+  n
```

Comment:
No the `n`
On Sun, Dec 20, 2015 at 9:25 AM Victor M. Suarez <notifications@github.com>
wrote:

> In src/accelerator.rs
> <https://github.com/calebmer/accelerate/pull/2#discussion_r48103873>:
>
> > @@ -0,0 +1,87 @@
> > +use drivers::Driver;
> > +use motions::Motion;
> > +
> > +fn clamp(n: isize, min: isize, max: isize) -> isize {
> > +  if n < min {
> > +    return min;
> > +  }
> > +  if n > max {
> > +    return max;
> > +  }
> > +  n
>
> clamp? it clamps....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/calebmer/accelerate/pull/2/files#r48103873>.
>